stress/shell/src/main/java/alluxio/stress/cli/fuse/FuseIOBench.java

```diff
@@ -293,23 +288,25 @@ private void runInternal() throws Exception {
       CommonUtils.sleepMs(waitMs);
       mStartBarrierPassed = true;
 
-      for (int i = 0; i < mFilesPath.size(); i++) {
-        mCurrentOffset = 0;
-        String filePath = mFilesPath.get(i);
-        while (!Thread.currentThread().isInterrupted()) {
-          if (isRead && CommonUtils.getCurrentMs() > mContext.getEndMs()) {
-            closeInStream();
-            return;
-          }
-          long ioBytes = applyOperation(filePath);
-
-          // Start recording after the warmup
-          if (CommonUtils.getCurrentMs() > recordMs) {
-            if (ioBytes > 0) {
-              mThreadCountResult.incrementIOBytes(ioBytes);
-            } else {
-              // Done reading/writing one file
-              break;
+      for (int dirId = mThreadId; dirId < mParameters.mNumDirs; dirId += mParameters.mThreads) {
+        for (int fileId = 0; fileId < mParameters.mNumFilesPerDir; fileId++) {
+          mCurrentOffset = 0;
+          String filePath = mParameters.mLocalPath + "/" + dirId + "/" + fileId;
```

Comment:
Avoid string concatenation, which is low efficiency. Each + will create a new object.
use string.format here, in other cases, can use stringBuilder